docs/README.md

```diff
@@ -8,7 +8,7 @@ The VTEX LocaleSwitcher is a VTEX component capable of changing the current lang
 
 ## Configuration
 
-1. Import the locale-switcher's app to your theme's dependencies in the `manifest.json`, for example:
+1. Import the `locale-switcher`'s app to your theme's dependencies in the `manifest.json`, for example:
 
```

Comment:
On line 18, add the following:::

Now, you are able to use all blocks exported by the `locale-switcher` app. Check out the full list below:

| Block name | Description | 
| --------  | ------------ | 
| `locale-switcher` | ![https://img.shields.io/badge/-Mandatory-red](https://img.shields.io/badge/-Mandatory-red) Basic block responsible for fetching server data to display language options for your store. | 
| `locale-switcher-list` | Should replace the `locale-switcher` block in scenarios where you want to display a custom Locale Switcher. Find more details on the Advanced Section below. |
| `locale-switcher-item`| Responsible for defining which language options will be available in a custom Locale Switcher. Find more details on the Advanced Section below. |